src/CGIHandler.cpp

```diff
@@ -1,8 +1,6 @@
 #include "CGIHandler.hpp"
-#include <vector>
-#include <sys/types.h>
-#include <sys/wait.h>
-#include <unistd.h>
+
+#define EXIT_FAILURE 1
```

Comment:
This define is still there.

Reply:
Sorry, must have got mixed up with the two branches - gone